src/Phinx/Migration/Manager.php

```diff
@@ -134,10 +134,11 @@ public function printStatus($environment, $format = null)
      *
      * @param string    $environment Environment
      * @param \DateTime $dateTime    Date to migrate to
+     * @param boolean   $dryRun      Output SQL rather than executing it
      *
      * @return void
      */
-    public function migrateToDateTime($environment, \DateTime $dateTime)
+    public function migrateToDateTime($environment, \DateTime $dateTime, $dryRun)
```

Comment:
Hmm, well... [the output is attached](https://github.com/robmorgan/phinx/blob/a601da707f00d40749fe8b2439c28368ed06daed/src/Phinx/Migration/Manager.php#L379-L399) to the manager, maybe it would make sense to attach the input as well?